### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by jenozin** [phanirithvij/twist.moe#27]
Hey, not an issue, just wanted to ask how do I access twist's content so I can download it? If I inspect the page I get the video URL is (for example) "https://twistcdn.bunny.sh/anime/onepiece/[HorribleSubs] One Piece - 011 [1080p].mp4"
But I can't access the video, I get 404 not found error. 

Thanks

**Comment by phanirithvij** [comment]
They are preventing downloading video using the url so you need to provide cookies and headers to let their server think a browser is requesting the video. That's what I'm doing in my script

**Comment by phanirithvij** [comment]
In short you can only do this programmatically not possible via normal tools so can't watch the video because they want us watch in the website itself and prevent others from uploading the anime on other websites.
Good thing twist.moe doesn't have ads as it's run on donations.

**Comment by jenozin** [comment]
> They are preventing downloading video using the url so you need to provide cookies and headers to let their server think a browser is requesting the video. That's what I'm doing in my script

Ah! I'll try to figure out how to do that! Thanks

**Comment by jenozin** [comment]
> In short you can only do this programmatically not possible via normal tools so can't watch the video because they want us watch in the website itself and prevent others from uploading the anime on other websites.
> Good thing twist.moe doesn't have ads as it's run on donations.

Yeah, makes sense. It is a great website, quality content. I was trying to get One Piece to watch during trips

**Comment by khalisafkari** [comment]
@jenozin You can add referrer in header

**Comment by Buchelaruzit** [comment]
> They are preventing downloading video using the url so you need to provide cookies and headers to let their server think a browser is requesting the video. That's what I'm doing in my script

Okay, and how do you do that? Sorry I don't know anything about all that stuff
